news/2025-05-20-numpy-2-migration-closure.md

```diff
@@ -0,0 +1,14 @@
+# Upcoming closure of NumPy 2.0 migration
+
+NumPy 2.0 was a big change (the first major version in 15 years). For more than a year, we
+have been migrating feedstocks from NumPy 1.x to NumPy 2.x, and while not every affected
+feedstock has been done, we are planning to conclude the migration at the end of May.
+
+For feedstocks that are not compatible with v2.x yet, this means you will have to add
```

Comment:
```suggestion
For feedstocks that are not compatible with v2.x yet, this means you will have to add
lines similar to
```

Reply:
I've made this more specific, so this shouldn't be necessary anymore.